src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/ColorExtension.shared.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+// https://developer.xamarin.com/samples/xamarin-forms/userinterface/animation/custom/
+namespace Xamarin.CommunityToolkit.Extensions
+{
+    public static class ColorExtension
+    {
+        public static Task<bool> ColorTo(this VisualElement self, Color fromColor, Color toColor, Action<Color> callback, uint length = 250, Easing easing = null)
```

Comment:
Can we change this to use ValueTask? Since we don't need to await anything? For the `ColorAnimationMethod` as well